python/hail/dataset.py

```diff
@@ -1534,12 +1527,14 @@ def filter_alleles(self, condition, annotation=None, subset=True, keep=True, fil
         :type condition: str
 
         :param annotation: Annotation modifying expression involving v (new variant), va (old variant annotations),
-            and aIndices (maps from new to old indices) (default: "va = va")
-        :param bool subset: If true, subsets the PL and AD, otherwise downcodes the PL and AD.
+            and aIndices (maps from new to old indices)
+            
+        :param bool subset: If true, subsets PL and AD, otherwise downcodes the PL and AD.
             Genotype and GQ are set based on the resulting PLs.
-        :param bool keep: Keep variants matching condition
-        :param bool filter_altered_genotypes: If set, any genotype call that would change due to filtering an allele
-            would be set to missing instead.
+            
+        :param bool keep: If true, keep variants matching condition
+        
+        :param bool filter_altered_genotypes: If true, genotypes that contain filtered-out alleles are set to missing.
```

Comment:
Could you please add the :return and :rtype while you have this pull request open?